pyciemss/interfaces.py

```diff
@@ -151,7 +182,12 @@ def wrapped_model():
         )()
 
         return prepare_interchange_dictionary(
```

Comment:
I would strongly prefer for us to not include this very specific output processing in the definition of `sample`. Instead, let's have the ensemble_quantiles be generated separately if needed from the existing output of `sample`.

Reply:
This way we can mix and match output formats if they change in the future.